alpha/nodejs-chart/README.md

```diff
@@ -0,0 +1,73 @@
+    kind: DockerImage
+    pushSecret: my-push-secret
+```
+
+## Values
+This section describes the Values used to configure this chart.
+
+Below is a table of each value used to configure this chart.
+
+| Value | Description | Default | Additional Information |
+| ----- | ----------- | ------- | ---------------------- |
+| `image.name` | Name of the image you want to build/deploy | Defaults to the Helm release name. | The chart will create/reference an [ImageStream](https://docs.openshift.com/container-platform/4.6/openshift_images/image-streams-manage.html) based on this value. |
+| `image.tag` | Tag that you want to build/deploy | `latest` | The chart will create/reference an [ImageStreamTag](https://docs.openshift.com/container-platform/4.6/openshift_images/image-streams-manage.html#images-using-imagestream-tags_image-streams-managing) based on the name provided |
+| `build.enabled` | Determines if build-related resources should be created. | `true` | Set this to `false` if you want to deploy a previously built image. Leave this set to `true` if you want to build and deploy a new image. |
+| `build.uri` | Git URI that references your git repo | https://github.com/nodeshift-starters/nodejs-rest-http | This value defaults to a sample application. Be sure to override this if you want to build and deploy your own application. |
+| `build.ref` | Git ref containing the application you want to build | main | - |
+| `build.contextDir` | The sub-directory where the application source code exists | - | - |
+| `build.output.kind` | Determines if the image will be pushed to an ImageStreamTag or a DockerImage (external registry) | ImageStreamTag | More information: More information: https://docs.openshift.com/container-platform/4.6/builds/managing-build-output.html |
+| `build.output.pushSecret` | Push secret name | - | Used only if build.output.kind == 'DockerImage' |
+| `build.pullSecret` | Image pull secret | - | More information: https://docs.openshift.com/container-platform/4.6/openshift_images/managing_images/using-image-pull-secrets.html |
+| `build.env` | Freeform `env` stanza | - | More information: https://kubernetes.io/docs/tasks/inject-data-application/define-environment-variable-container/ |
+| `build.resources` | Freeform `resources` stanza | - | More information: https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/ |
+| `deploy.replicas` | Number of pod replicas to deploy | `1` | - |
+| `deploy.resources` | Freeform `resources` stanza | - | More information: https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/ |
+| `deploy.serviceType` | Type of service to create | `ClusterIP` | More information: https://kubernetes.io/docs/concepts/services-networking/service/#publishing-services-service-types |
+| `deploy.ports` | Freeform service `ports` stanza. | See [values.yaml](./values.yaml) | More information: https://kubernetes.io/docs/concepts/services-networking/service/#defining-a-service |
```

Comment:
I'm on the fence whether a reference to the values.yaml versus just including the default info in-line

Reply:
I felt that less-technical users would find it helpful in the readme, but get what you're saying if you're trying to avoid duplicating the reference in values.yaml and readme.

Reply:
I was thinking it might make sense to pull in versus the reference to the values.yaml, but don't feel to strongly one way or the other.